assets/Product.wxs

```diff
@@ -259,7 +259,7 @@
                 <RegistryValue Type="string" Name="MUIVerb" Value="$(var.PowerShellFileContextSubMenuDialogText)"/>
                 <RegistryValue Type="string" Name="Icon" Value="[VersionFolder]pwsh.exe" KeyPath="yes"/>
                 <RegistryKey Key="Command">
-                  <RegistryValue Type="string" Value="[VersionFolder]pwsh.exe -Command &quot;$host.UI.RawUI.WindowTitle = '$(var.ProductName) $(var.SimpleProductVersion) ($(sys.BUILDARCH))'; if((Get-ExecutionPolicy ) -ne 'AllSigned') { Set-ExecutionPolicy -Scope Process Bypass }; &amp; '%1'&quot;"/>
+                  <RegistryValue Type="string" Value="[VersionFolder]pwsh.exe -file &quot; '%1'&quot;"/>
```

Comment:
I think you don't need the inner single quotes.  The `&quot;` will add a double quote and take care of whitespace in the path

```suggestion
                  <RegistryValue Type="string" Value="[VersionFolder]pwsh.exe -file &quot;%1&quot;"/>
```